src/main/java/io/github/mzmine/modules/io/import_rawdata_mzml/msdk/data/MzMLParser.java

```diff
@@ -53,42 +51,27 @@
  */
 public class MzMLParser {
 
-  private Vars vars;
-  private TagTracker tracker;
-  private final MzMLRawDataFile newRawFile;
-  private final MzMLFileImportMethod importer;
-  private int totalScans = 0, parsedScans = 0;
   private static final Logger logger = Logger.getLogger(MzMLParser.class.getName());
 
-  private ParameterSet advancedParameters;
+  private final Vars vars;
```

Comment:
maybe rename

Reply:
this Vars thing is super weird... but I would just keep it for now. We can rename stuff later